packages/suite/src/components/suite/AccountStickyContent/index.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FiatValue, FormattedCryptoAmount, AccountLabeling, Translation } from '@suite-components';
+import { CoinLogo, variables } from '@trezor/components';
+import { Account } from '@wallet-types';
+import { Route } from '@suite-constants/routes';
+
+const Main = styled.div`
+    align-items: center;
+    display: flex;
+`;
+
+const BalanceWrapperContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    margin: 0 0 0 13px;
+`;
+
+const Balance = styled.div`
+    white-space: nowrap;
+    font-size: ${variables.FONT_SIZE.SMALL};
+`;
+
+const FiatBalanceWrapper = styled.div`
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+    margin-left: 0.5ch;
+`;
+
+const LabelWrapper = styled.div`
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    font-size: ${variables.FONT_SIZE.NORMAL};
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    margin: 0 13px 0 0;
+`;
+
+const BalanceInner = styled.div`
+    display: flex;
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+const StyledFiatValue = styled.div`
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+const StyledFormTitle = styled.h3`
+    font-size: ${variables.FONT_SIZE.NORMAL};
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+`;
+
+const StyledSeparator = styled.span`
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+    font-size: 31px;
+    font-family: initial;
+    display: inline-block;
+    line-height: 1;
+    padding: 0 8px;
+`;
+
+interface Props {
+    account?: Account;
+    routeName: Route['name'];
+}
+
+const AccountStickyContent = (props: Props) => {
+    const { account, routeName } = props;
+
+    if (!account) return null;
+
+    const { symbol, formattedBalance } = account;
+
+    const headerText = () => {
+        if (routeName && routeName === 'wallet-send') {
+            return <Translation id="TR_NAV_SEND" />;
+        }
+        if (routeName && routeName === 'wallet-receive') {
+            return <Translation id="TR_NAV_RECEIVE" />;
+        }
+        if (routeName && routeName.startsWith('wallet-coinmarket')) {
+            return <Translation id="TR_NAV_TRADE" />;
+        }
+    };
```

Comment:
totally optional nit:
```suggestion
    const headerText = () => {
        if (!routeName) return;
        
        if (routeName === 'wallet-send') {
            return <Translation id="TR_NAV_SEND" />;
        }
        if (routeName === 'wallet-receive') {
            return <Translation id="TR_NAV_RECEIVE" />;
        }
        if (routeName.startsWith('wallet-coinmarket')) {
            return <Translation id="TR_NAV_TRADE" />;
        }
    };
```

Reply:
eb9750cc134ed0fb8ec75d435421489ad4f796fc